tests/integration/components/flexberry-dropdown-test.js

```diff
@@ -0,0 +1,612 @@
+
+      clickOnDropdown(assert, resolve, $dropdomnItem, null, null);
+    });
+
+    // Wait animation to be completed (when resolve will be called inside previous timeout).
+    // Then try to collapse component.
+    itemAnimationCompleted.then(() => {
+      // Wait for collapse animation to be completed & check component's state.
+      Ember.run(() => {
+        let animationCompleted = assert.async();
+        setTimeout(() => {
+          let $dropdownChangeText = $component.children('div.text');
+          let $dropdownActiveItem = $dropdownMenu.children('div.item.active');
+          let dropdownActiveItemText = $dropdownActiveItem.text();
+
+          // Check that component is expanded now.
+          assert.strictEqual($dropdownActiveItem.size(), 1, 'Only one component\'s item is active');
+          assert.strictEqual(Ember.$.trim($dropdownChangeText.text()), dropdownActiveItemText, 'Component\'s default text is changes');
+          assert.strictEqual(this.get('value'), dropdownActiveItemText,
+          'Component\'s property binded to \'value\' is equals to \'' + dropdownActiveItemText + '\'');
+          animationCompleted();
+        }, animationDuration);
+      });
+    });
+  });
+});
+
+test('changes in property binded to \'value\' causes changes in <dropdown>', function(assert) {
+  assert.expect(7);
+
+  // Create array for testing.
+  let itemsArray = ['Caption1', 'Caption2', 'Caption3'];
+  this.set('itemsArray', itemsArray);
+  this.set('value', null);
+
+  // Render component.
+  this.render(hbs`{{flexberry-dropdown
+    value=value
+    items=itemsArray
+  }}`);
+
+  // Retrieve component.
+  let $component = this.$().children();
+  let $dropdownMenu = $component.children('div.menu');
+  let $dropdownText = $component.children('div.text');
+  let $items = Ember.$('div.item', $dropdownMenu);
+
+  // Check <dropdown>'s value & binded value for initial emptyness.
+  assert.strictEqual($dropdownText.hasClass('default'), true, 'Component\'s text has class \'default\'');
+  assert.strictEqual(Ember.$.trim($dropdownText.text()), Ember.get(I18nRuLocale, 'components.flexberry-dropdown.placeholder'),
+  'Component\'s inner <dropdown\'s value is equals to \'\'');
+  assert.strictEqual(this.get('value'), null, 'Component\'s property binded to \'value\' is equals to null');
+
+  // Change property binded to 'value' & check them again.
+  let newValue = itemsArray[0];
+  this.set('value', newValue);
+  let dropdownChangeText = Ember.$($items[0]).text();
+
+  assert.strictEqual(this.get('value'), newValue, 'Component\'s property binded to \'value\' is equals to \'' + newValue + '\'');
+  assert.strictEqual(dropdownChangeText, newValue, 'Component\'s inner <dropdown>\'s value is equals to \'' + newValue + '\'');
+  assert.strictEqual($dropdownText.hasClass('default text'), false, 'Component\'s text hasn\'t class \'default\'');
+  assert.strictEqual($dropdownText.hasClass('text'), true, 'Component\'s text has class \'text\'');
```

Comment:
Instead of this assertions, yuo should select div.active.selected, check that only one item is selected, & check it's $selectedItem.text() for equality to newValue.
Also select $component.children('div.text') & perform same check for equality to new value, also check that it hasn't class 'default'.